.gitignore

```diff
@@ -123,3 +130,9 @@ venv.bak/
 # ignore .nur files in NuRadioReco/examples/
 # this should not apply for NuRadioReco/examples/example_data/
 NuRadioReco/examples/*.nur
+
+# ignore test output (not include input or reference files here!)
+*output*.nur
+*output*.hdf5
```

Comment:
I don't like this wildcard. We might want to add files matching this string later. It will be difficult to find out for developers why they don't show up in git. I think we need to go though the extra effort of adding the exact file names. 
Or what do other people think? This of course is an additional safeguard against adding larger files by mistake.... but sometimes we want to add new nur files which would then always require to update the global gitignore file first.

Reply:
just to mention it. I explicitly exclude from being ignored those:
```
!*reference*.nur
!*reference*.hdf5
```
My intention was: Its annoying to maintain gitignore files. If we come up with a naming scheme for our input, output, reference files we could use wildcards and also remove quite some local gitignores. 

That being said I can add explicit entries again.

Reply:
Yes I saw that. But it is a very hidden implicit convention for a naming scheme. I think there a pro and cons to both options. Let's discuss it in one of the next inicemc calls.